src/uv.cc

```diff
@@ -60,15 +60,6 @@ using v8::Value;
 
 void ErrName(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args);
-  if (env->options()->pending_deprecation && env->EmitErrNameWarning()) {
```

Comment:
Why is this deprecation message being removed without being replaced elsewhere?

Reply:
Yeah, It's being moved to JS land on this PR: https://github.com/nodejs/node/pull/30816, I was considering mix this PR and the other with this on JS land.